### DISCUSSION TITLE
Recent update and vtsls no longer respects disabling JS/TS validation?

### DISCUSSION THREAD

**Discussion by alexciarlillo** [LazyVim/LazyVim#4430] [Q&A] [answered]
I recently took a handful of updates and I am unsure what broke this. I use vtsls LSP on a large monorepo Javascript project and have disabled TS validation - but it is now back.

My config:

```
{
    "neovim/nvim-lspconfig",
    opts = {
      inlay_hints = { enabled = false },
      servers = {
        vtsls = {
          root_dir = function()
            local lazyvimRoot = require("lazyvim.util.root")
            return lazyvimRoot.git()
          end,
          autoUseWorkspaceTsdk = true,
          settings = {
            typescript = {
              tsserver = {
                -- log = "verbose",
                maxTsServerMemory = 4096,
                watchOptions = {
                  watchFile = "useFsEventsOnParentDirectory",
                  fallbackPolling = "dynamicPriorityPolling",
                },
              },
              disableAutomaticTypeAcquisition = true,
              validate = {
                enable = false,
              },
            },
          },
        },
      },
    },
```

And here is an inline validation error that is now appearing:
![image](https://github.com/user-attachments/assets/62799e28-668b-479a-b05b-c09626470c99)

Did something potentially change with the config structure?

**Comment by alexciarlillo** [comment, marked as answer]
Ahh apparently it has to do with [this](https://github.com/neovim/nvim-lspconfig/pull/3232) 

I was able to fix by adding:

```
ts_ls = {
   enabled = false
}
```

**Reply by RHOOPH** [reply]
Thanks, 

This update also caused vtsls and tsserver to be running on the same buffer. 

![image](https://github.com/user-attachments/assets/f444824f-a968-4ea2-960a-40116a939dc1)


This config needs to be changed to disable ts_ls instead of tsserver. 
https://www.lazyvim.org/extras/lang/typescript#nvim-lspconfig

```lua
return {
  {
    "neovim/nvim-lspconfig",
    opts = {
      servers = {
        ts_ls = {
          enabled = false,
        },
      },
    },
  },
}
```

**Reply by dpetka2001** [reply]
There's already #4406 about that.
